src/protocol-parser/webdriver-bidi.ts

```diff
@@ -108,7 +108,9 @@ export const ErrorCodeSchema = z.lazy(() =>
     'no such element',
```

Comment:
Like discussed offline I think we should create a separate file next to this one with the types generated from the CDDL for the Network Interception PR.

Reply:
Not applicable anymore, given that the upstream PR was merged.